<?xml version="1.0" ?>
<launch>
	<node pkg="joy" type="joy_node" name="joy_node">
		<param name="dev" type="string" value="/dev/input/js0" />
		<param name="deadzone" value="0.12" />
	</node>
	
	<!-- start youbot teleop node -->
	<node pkg="youbot_teleop" type="teleop_joypad_node" name="youbot_teleop_joypad" output="screen">
		<param name="max_linear_x_speed" value="0.3" />
		<param name="max_linear_y_speed" value="0.3" />
		<param name="max_angular_speed" value="0.5" />
	</node>
</launch>
